.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 sudo: false
 node_js:
+  - "8"
```

Comment:
Why "8" and "6" but not "7"? Does order here matter?

Reply:
I don't think order matters for our purposes.  It probably matters at some like, like Travis spawns the jobs in the order you list them.

Even number Node releases become LTS, odd numbered never provide support.  Effectively, odd numbered releases are unstable and experimental.  6 is currently LTS, and 8 will become LTS in October.  Typically, once a newer even numbered release is available, we've stopped testing the odd numbered release before it, since there's no expectation of support.

Did that make sense?  Still seem reasonable?

Reply:
Thank you for your explanation. TIL :point_up:

Reply:
No prob.  Here's more info on the node release process/schedule: https://github.com/nodejs/Release